### PR TITLE
nigeria_ebonyi_state: remove tab characters

### LINKED ISSUE #901: nigeria_ebonyi_state: Contains literal TAB character `\t`, which is invalid JSON
Like https://github.com/open-contracting/kingfisher-collect/pull/878, we should replace `'\t'` with `r'\t'` in the response text.

### PR DESCRIPTION
**Pull request by Ravf95**
Closes #901

### CODE REVIEW

**Review by yolile — DISMISSED** [review]

File: kingfisher_scrapy/spiders/nigeria_ebonyi_state.py [hunk 1]
```diff
@@ -26,3 +26,11 @@ class NigeriaEbonyiState(PeriodicSpider):
 
     # SimpleSpider
     data_type = 'release_package'
+
+    @handle_http_error
+    def parse(self, response):
+        data = response.text
+        # Remove tab characters to prevent invalid JSON errors within strings
+        data = data.replace('\t', '')
```
Comment (jpmckinney) [inline kingfisher_scrapy/spiders/nigeria_ebonyi_state.py:34]: Shouldn't we at least replace it with a single space?

Comment (Ravf95) [reply kingfisher_scrapy/spiders/nigeria_ebonyi_state.py:34]: Yes, we can replace it with a single space, I will change it.

File: kingfisher_scrapy/spiders/nigeria_ebonyi_state.py [hunk 2]
```diff
@@ -29,8 +29,11 @@ class NigeriaEbonyiState(PeriodicSpider):
 
     @handle_http_error
     def parse(self, response):
+        '''
+        The JSON data sometimes contains unescaped tab characters within strings.
+        '''
```
Comment (jpmckinney) [inline kingfisher_scrapy/spiders/nigeria_ebonyi_state.py:34]: I meant the main docstring in a Caveats section like https://kingfisher-collect.readthedocs.io/en/latest/spiders.html#armenia

Comment (Ravf95) [reply kingfisher_scrapy/spiders/nigeria_ebonyi_state.py]: Sorry, I move it in caveats section.

**Review by jpmckinney — APPROVED** [review]

**Comment by yolile** [comment]
deployed in both, kingfisher and the registry

**Pull request merged**